docs/features/UNIFIED_CANVAS.md

```diff
@@ -0,0 +1,96 @@
+The Unified Canvas is a tool designed to facilitate composition, enabling artists to use all available generation modes in a single unified workflow. This documentation will help explain the basics of using the Unified Canvas, as well as some advanced tools available to power users of the canvas.
+
+# Basics
+The Unified Canvas consists of two layers: The **Base Layer** and the **Mask Layer**, which can be toggled using the (Q) hotkey.
+
+### Base Layer
+The **Base Layer** is the image content currently managed by the canvas, and can be exported at any time to the gallery by using the **Save to Gallery** option. When the Base Layer is selected, the Brush (B) and Eraser (E) tools will directly manipulate the base layer. Any images uploaded to the canvas (or sent to the unified canvas from the gallery) will reset the canvas, and add the image as your initial base layer image.
+
+### Staging Area
+When you generate images, they will display in the canvas **Staging Area**, alongside the Staging Area toolbar buttons. While the Staging Area is active, you cannot interact with the canvas.
+
+Accepting generations will commit the new generation to the **Base Layer**. You can review all generated images using the Prev/Next arrows, save any individual generations to your gallery (without committing to the Base layer) or discard generations. While you can Undo a discard in an individual Canvas session, any generations that are not saved will be lost when the canvas resets.
+
+### Mask Layer
+The **Mask Layer** consists of any masks that have been created to inform Inpainting generations. You can generate a new mask by using the Brush tool with the Mask layer set as your Active layer. Masks will only be applied inside of the current bounding box.
+
+### Bounding Box
+When generating a new image, Invoke will process and apply new images within the area denoted by the **Bounding Box**. New invocations will be generated and applied based on the Width & Height settings of the Bounding Box. The Bounding Box can be moved and resized using the Move (V) tool - It can also be resized using the Bounding Box options in the Options Panel.
+
+### Inpainting & Outpainting
+"Inpainting" means asking the AI to refine part of an image while leaving the rest alone. For example, updating a portrait of your grandmother to have her wear a biker's jacket.
+"Outpainting" means asking the AI to expand the original image beyond its original borders, making a bigger image that's still based on the original. For example, extending the above image of your Grandmother in a biker's jacket to include her wearing biker pants (and while we're at it, a motorcycle!)
```

Comment:
Great example. If you have time, you could add an example image to spice things up.